scipy/stats/tests/test_distributions.py

```diff
@@ -371,6 +371,195 @@ def test_pdf_zero(self):
+        np.random.seed(1234)
+
+    def test_pdf_r(self):
+        # test against R package GeneralizedHyperbolic
+        # x <- seq(-10, 10, length.out = 10)
+        # GeneralizedHyperbolic::dghyp(
+        #    x = x, lambda = 2, alpha = 2, beta = 1, delta = 1.5, mu = 0.5
+        # )
+        vals_R = np.array([
+            2.94895678275316e-13, 1.75746848647696e-10, 9.48149804073045e-08,
+            4.17862521692026e-05, 0.0103947630463822, 0.240864958986839,
+            0.162833527161649, 0.0374609592899472, 0.00634894847327781,
+            0.000941920705790324
+            ])
+
+        lmbda = 2
+        alpha = 2
+        beta = 1
+        delta = 1.5
+        mu = 0.5
+
+        alpha_hat = alpha * delta
+        beta_hat = beta * delta
+
+        x = np.linspace(-10, 10, 10)
+        gh = stats.genhyperbolic(
+            p=lmbda,
+            a=alpha_hat,
+            b=beta_hat,
+            loc=mu,
+            scale=delta
+            )
+
+        assert_allclose(gh.pdf(x), vals_R, atol=1e-9)
+
+    def test_cdf_r(self):
+        # test against R package GeneralizedHyperbolic
+        # q <- seq(-10, 10, length.out = 10)
+        # GeneralizedHyperbolic::pghyp(
+        #   q = q, lambda = 2, alpha = 2, beta = 1, delta = 1.5, mu = 0.5
+        # )
+        vals_R = np.array([
+            1.01881590921421e-13, 6.13697274983578e-11, 3.37504977637992e-08,
+            1.55258698166181e-05, 0.00447005453832497, 0.228935323956347,
+            0.755759458895243, 0.953061062884484, 0.992598013917513,
+            0.998942646586662
+            ])
+
+        lmbda = 2
+        alpha = 2
+        beta = 1
+        delta = 1.5
+        mu = 0.5
+
+        alpha_hat = alpha * delta
+        beta_hat = beta * delta
+
+        x = np.linspace(-10, 10, 10)
+        gh = stats.genhyperbolic(
+            p=lmbda,
+            a=alpha_hat,
+            b=beta_hat,
+            loc=mu,
+            scale=delta
+            )
+
+        assert_allclose(gh.cdf(x), vals_R, atol=1e-9, rtol=1e-5)
+
+    def test_moments_r(self):
+        # test against R package GeneralizedHyperbolic
+        # sapply(1:4,
+        #    function(x) GeneralizedHyperbolic::ghypMom(
+        #        order = x, lambda = 2, alpha = 2,
+        #        beta = 1, delta = 1.5, mu = 0.5,
+        #        momType = 'raw')
+        # )
+
+        vals_R = np.array([
+            2.36848366948115, 8.4739346779246,
+            37.8870502710066, 205.76608511485
+            ])
+
+        lmbda = 2
+        alpha = 2
+        beta = 1
+        delta = 1.5
+        mu = 0.5
+
+        alpha_hat = alpha * delta
+        beta_hat = beta * delta
+
+        vals_us = [
+            stats.genhyperbolic(p=lmbda, a=alpha_hat, b=beta_hat,
+                                loc=mu, scale=delta).moment(i)
+            for i in range(1, 5)
+            ]
+
+        assert_allclose(vals_us, vals_R, atol=1e-9, rtol=1e-8)
+
+    def test_rvs(self):
+        # KS test
```

Comment:
is this just an additional test compared to the one with the default parameters? maybe replace "KS test" by a longer description

Reply:
new comment says `# Kolmogorov-Smirnov test to ensure alignemnt of analitycal and empirical cdfs` (btw: typo in analytical ;)
my question was mainly whether this is a special edge case, e.g. because beta = 0 or just a random test case in addition to the default parameters in the file `_distr_params.py`. note that for those parameters, a KS test is performed in the default tests that are run for every distribution.

Reply:
You are correct, the test was included before i realized we had a KS test in the common tests. So it is redundant. Will fix typo, while we decide whether to leave it there or not.

Reply:
I'm happy to leave it.